src/builder.rs

```diff
@@ -315,7 +322,7 @@ impl StatsdBuilder {
             ExporterConfig::Unconfigured => Err(BuildError::MissingExporterConfiguration),
             ExporterConfig::PushGateway { endpoint, interval } => {
                 let exporter = async move {
-                    let client = UdpSocket::bind("0.0.0.0:0").await?;
+                    let client = UdpSocket::bind("[::]:0").await?;
```

Comment:
TIL this does a dual-stack bind by default

Reply:
I learned this myself when investigating this :)

Note, I am not going to merge this into our main branch here. since I have the PR open from my branch into the upstream repo to.